src/gps/gps_util.h

```diff
@@ -0,0 +1,80 @@
+#pragma once
+
+#include "../simulator/utils.h"
+
+// Represents a GPS coordinate in degrees. Positive is north/east.
+struct gpscoords_t {
+	// the latitude of the gps coordinate, in degrees
+	double lat;
+	// the longitude of the gps coordinate, in degrees
+	double lon;
+};
```

Comment:
Could you convert these to Doxygen comments?
```suggestion
/** Represents a GPS coordinate in degrees. Positive is north/east. */
struct gpscoords_t {
	/** the latitude of the gps coordinate, in degrees */
	double lat;
	/** the longitude of the gps coordinate, in degrees */
	double lon;
};
```